test/libsolidity/syntaxTests/pragma/unterminated_version.sol

```diff
@@ -1,4 +1,4 @@
 pragma solidity 0.4.3
 pragma abicoder v2;
 // ----
-// ParserError 1684: (0-41): Found version pragma, but failed to parse it. Please ensure there is a trailing semicolon.
```

Comment:
Here, the suggestion would be quite useful. Why did you remove it?

Reply:
Because it's not always that reason. At least to me this reason sounds confusing when the syntax error is actually way before the `;`. I can drop that single commit though.